### PR TITLE
Create a page for LSP14OwnableStep

### PR DESCRIPTION
**Pull request by b00ste**
## What does this PR introduce?

This PR introduces documentation about The **LSP14Ownable2Step** and modifies the current mentions of **ClaimOwnership**.

- There is a new page added in the _Smart Contract ABI_ section and in the _Generic Standards_ section.
- The previous mentions of **ClaimOwnership** in _LSP0ERC725Account_ and in _LSP9Vault_ are modified according to the latest updates of the LSP0 and LSP9 contracts that now use LSP14.

### CODE REVIEW

**Comment by Hugoo** [comment]
Package lock is not needed we only use yarn in this repo

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+:::info Standard Document
+
+[LSP14 - Ownable 2-Step](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-14-Ownable2Step.md)
+
+:::
+
+## Introduction
+
+The **[LSP14 Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-14-Ownable2Step.md)** describes an extended version of **[EIP173](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-173.md)** that uses a 2-step process to transfer or renounce ownership of a contract, instead of instant execution.
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:16]: This is the same content as what is in the Introduction section of the LSP14 standards specs.

I don't think we should have exactly the same content as the standards specs, but rather a short and layman version. A good headline would be:

- _what is the existing problem of the Ownable standard?_
- _what is the proposed solution using LSP14?_

And in the section **## What does this standard represent ?** I would describe:

_How LSP14 solves it._

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+In addition, this standard defines hooks that call the **[universalReceiver(...)](../smart-contracts/lsp0-erc725-account.md#universalreceiver)** function of the current owner and new owner, if these addresses are contracts that implement LSP1. This aims to:
+
+- notify when the new owner of the contract should accept ownership.
+
+![Transfer Ownership Notification](/img/standards/lsp14/transfer-ownership-notification.jpeg)
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:22]: drawings and illustrations are great 👌 

I think this should not be the first thing in the Standard page. There should be a separate section about **notification hooks**.

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+- notify the previous and new owner when ownership of the contract has been fully transferred.
+
+![Accept Ownership Notification](/img/standards/lsp14/accept-ownership-notification.jpeg)
+
+## What does this standard represent ?
+
+**LSP14 - Ownable2Step** is a modified version of [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) that uses a safer mechanism for transferring and renouncing ownership.
+
+In EIP173, ownership of a contract is transferred directly to a new owner, potentially leading to blocking access to the contract. For instance, if the owner calls `transferOwnership()` and the new owner:
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:32]: All of this content here in this section, starting from this paragraph up to _"making the processes of transferring and renouncing ownership a 2-step process."_ would fit better in the Introduction.

Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:32]:
```suggestion
In the current `Ownable` standard (EIP173), ownership of a contract is transferred directly in one transaction via `transferOwnership(...)`. This presents some risks. For instance if the new owner:
```

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+- is an EOA that lost its private key.
+- is an `address` entered incorrectly.
+
+The same thing happens with renouncing ownership of the contract, it is done in a single transaction. This can lead to renouncing ownership of the contract by accident and potentially losing access to the contract.
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:37]:
```suggestion
Renouncing ownership of the contract in `Ownable` is also done in one transaction. If the owner accidentally calls `renounceOwnership(...)`, this leads to losing access to the contract.
```

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+With **LSP14 - Ownable2Step**, those two problem are tackled by making the processes of transferring and renouncing ownership a 2-step process.
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:39]:
```suggestion
**LSP14 - Ownable2Step** tackles these two problems using a safer mechanism for managing contract ownership. Transferring and renouncing ownership now works as a 2-step process.
```

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+![Transfer Ownership Notification](/img/standards/lsp14/transfer-ownership-notification.jpeg)
+
+- notify the previous and new owner when ownership of the contract has been fully transferred.
+
+![Accept Ownership Notification](/img/standards/lsp14/accept-ownership-notification.jpeg)
+
+## What does this standard represent ?
+
+**LSP14 - Ownable2Step** is a modified version of [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) that uses a safer mechanism for transferring and renouncing ownership.
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:30]: I suggest to merge this sentence at the bottom, in my suggestion below.

**Review by CJ42 — COMMENTED** [review]
added review comment

File: docs/standards/generic-standards/lsp14-ownable-2-step.md (hunk 6 above)
Comment (b00ste) [reply docs/standards/generic-standards/lsp14-ownable-2-step.md]: I did not understand this comment, let me know if it is still the case

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+:::info Standard Document
+
+[LSP14 - Ownable 2-Step](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-14-Ownable2Step.md)
+
+:::
+
+## Introduction
+
+**LSP14 - Ownable2Step** is a modified version of [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) that uses a safer mechanism for transferring and renouncing ownership.
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:16]: Please move this paragraph below under the following section. As follow:

---

## What does this standard represent ?

**LSP14 - Ownable2Step** is an extended version of [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) that uses a safer mechanism for transferring and renouncing ownership. It modifies the processes of _tranferring and renouncing ownership_ in the following way:

**Review by CJ42 — CHANGES_REQUESTED** [review]

**Review by YamenMerhi — CHANGES_REQUESTED** [review]
First Review of the Standard Page:
Great Job! Just we need to agree on the suggested changes @CJ42 @b00ste. Let's discuss internally.

- [ ] Introduction should be general and not start describing LSP14
- [ ] I would suggest to stop naming the pending owner as newOwner as it could be confusing.

If we want to have a figure like this to explain we can start refer to them as address A and address B because current owner on step 1 will not be current owner on step 2. It will be the previous owner. Same as newOwner, in step 1 he's still the pending owner but in step 2 he is the new owner. This could be a lot simplified if we go with address A or Person A or like Alice or Bob. 

Screen Shot 2022-10-03 at 3 56 29 PM

- [x] Ownable shouldn't be mentioned as this is the name of contract implementation not the standard.
- [x] Renounce Ownership section should come last not between transferOwnership and accept ownership hooks.

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+In the current `Ownable` standard (EIP173), ownership of a contract is transferred directly in one transaction via `transferOwnership(...)`. This presents some risks. For instance if the new owner:
+
+- is an EOA that lost its private key.
+- is an `address` entered incorrectly.
+
+Renouncing ownership of the contract in `Ownable` is also done in one transaction. If the owner accidentally calls `renounceOwnership()`, this leads to losing access to the contract.
+
+**LSP14 - Ownable2Step** tackles these two problems using a safer mechanism for managing contract ownership. Transferring and renouncing ownership now works as a 2-step process.
```
[the remark this answers is not in the document]
Comment (b00ste) [reply docs/standards/generic-standards/lsp14-ownable-2-step.md:25]: The paragraph at line 23  is a continuation of the first paragraph of the Introduction and the paragraph at line 25 is describing a way of solving the problems.

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+- is an EOA that lost its private key.
+- is an `address` entered incorrectly.
+
+Renouncing ownership of the contract in [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) is also done in one transaction. If the owner accidentally calls `renounceOwnership()`, this leads to losing access to the contract.
+
+**LSP14 - Ownable2Step** tackles these two problems using a safer mechanism for managing contract ownership. Transferring and renouncing ownership now works as a 2-step process.
+
+## What does this standard represent ?
+
+**LSP14 - Ownable2Step** is a modified version of [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) that uses a safer mechanism for transferring and renouncing ownership.
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:27]:
```suggestion
**LSP14 - Ownable2Step** is an extended version of [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) that uses a 2-step process for transferring and renouncing ownership.
```

File: docs/standards/generic-standards/lsp14-ownable-2-step.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+## Introduction
+
+In the current [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) standard (EIP173), ownership of a contract is transferred directly in one transaction via `transferOwnership(...)`. This presents some risks. For instance if the new owner:
+
+- is an EOA that lost its private key.
+- is an `address` entered incorrectly.
+
+Renouncing ownership of the contract in [EIP173 - Contract Ownership Standard](https://eips.ethereum.org/EIPS/eip-173) is also done in one transaction. If the owner accidentally calls `renounceOwnership()`, this leads to losing access to the contract.
+
+**LSP14 - Ownable2Step** tackles these two problems using a safer mechanism for managing contract ownership. Transferring and renouncing ownership now works as a 2-step process.
```
Comment (CJ42) [inline docs/standards/generic-standards/lsp14-ownable-2-step.md:23]: Keep it more generic to introduce the solution proposed by LSP14.

```suggestion
What is needed is a safer mechanism for managing contract ownership.
```

**Review by YamenMerhi — APPROVED** [review]

**Pull request merged**